packages/whenable/src/tools.js

```diff
@@ -0,0 +1,33 @@
+/* global globalThis */
+// @ts-check
+import { makeWhen } from './when.js';
+import { prepareWhenableKits } from './whenable.js';
+import { prepareWatch } from './watch.js';
+import makeE from './E.js';
+
+/**
+ * @param {import('@agoric/base-zone').Zone} zone
+ * @param {object} [powers]
+ * @param {(reason: any) => boolean} [powers.rejectionMeansRetry]
+ * @param {(p: PromiseLike<any>, watcher: import('./watch.js').PromiseWatcher, ...args: unknown[]) => void} [powers.watchPromise]
+ */
+export const prepareWhenableTools = (zone, powers) => {
+  const { rejectionMeansRetry = () => false, watchPromise } = powers || {};
+  const { makeWhenableKit, makeWhenablePromiseKit } = prepareWhenableKits(zone);
+  const when = makeWhen(makeWhenablePromiseKit, rejectionMeansRetry);
+  const watch = prepareWatch(
+    zone,
+    makeWhenableKit,
+    watchPromise,
+    rejectionMeansRetry,
+  );
+  const E = makeE(
+    globalThis.HandledPromise,
+    harden({
+      unwrap: when,
+      additional: { watch },
+    }),
+  );
+  return harden({ E, when, watch, makeWhenableKit });
```

Comment:
```suggestion
  return harden({ E, makeWhenableKit });
```
or actually now
```suggestion
  return harden({ E, makeVowKit });
```



`index.js exports `* from 'tools.js'`, so this really is the defining normal use API. I see that the new `E` already has the one-arg `E.when(x)` behavior, so there's already no need to include `when` here. Please also add `E.watch` and do not include `watch` here.